src/renderers/webgl/WebGLRenderStates.js

```diff
@@ -54,26 +61,26 @@ function WebGLRenderStates() {
 
 	let renderStates = new WeakMap();
 
-	function get( scene, camera ) {
+	function get( scene, renderCallDepth ) {
```

Comment:
Let's use a default parameter here:
```js
function get( scene, renderCallDepth = 0 ) {
```
This will make `renderCallDepth` optional in `WebGLRenderStates.d.ts`. And this will simplify the code in `WebGLRenderer.compile()` to `currentRenderState = renderStates.get( scene );`

Reply:
Done!